File: Python/utils/path_utils.py
class PathUtils:
    @staticmethod
    def save_path_to_file(path, filename):
        with open(filename, "w") as f:
            for x, y in path:
                f.write(f"{x},{y}\n")
        print(f"Path saved to {filename}")

    @staticmethod
    def load_path_from_file(filename):
        path = []
        with open(filename, "r") as f:
            for line in f:
                x_str, y_str = line.strip().split(",")
                path.append((int(x_str), int(y_str)))
        print(f"Path loaded from {filename}")
        return path

File: C_SHARP/PathUtils.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Runtime.CompilerServices;

namespace OpenNPC_CSharp_ver
{
	public class PathUtils
	{
		public static void SavePath(List<(int, int)> path, string filename)
		{
			using (var writer = new StreamWriter(filename))
			{
				foreach (var (x, y) in path)
				{
					writer.WriteLine($"{x},{y}");
				}
			}
			Console.WriteLine($"Path saved to {filename}");
		}

		public static List<(int, int)> LoadPath(string filename)
		{
			var path = new List<(int, int)> ();
			foreach (var line in File.ReadAllLines(filename))
			{
				var parts = line.Split('.');
				int x = int.Parse(parts[0]);
				int y = int.Parse(parts[1]);
				path.Add((x, y));
			}
			Console.WriteLine($"Path loaded from {filename}");
			return path;
		}
	}
}
